### PR TITLE
Banter updates for 2016 school year

### PR DESCRIPTION
**Pull request by skier31415**

### CODE REVIEW

File: resources/banter/index.html [hunk 1]
```diff
@@ -62,76 +62,61 @@
...
             <div class="col-sm-12">
-              <h2>Introduction</h2>
-              <h3>Banter is…</h3>
-              <p>an Olin alumni-run program designed to connect current students and alumni to share experience. Newly expanded to connect alumni with alumni !</p>
-              <h4>Banter</h4>
-              <p>noun.</p>
-              <ul>
-                <li>a chance to build a lasting relationship with a member of the Olin alumni community, while receiving advice such as:
+              <h2>Banter</h2>
+              <h4>TL;DR.</h4><p>Meet an <strong>Olin alumnus</strong>. Find an <strong>awesome mentor</strong>. Discuss life at Olin, how to adult, what you did last week, and everything in between. Hindsight is 20/20! Borrow some perspective from someone who's been there.</p>
```
Comment (gregmarra) [inline resources/banter/index.html]: Can you put each block level tag on its own line? In this case, linebreak after the `</h4>`.

File: resources/banter/index.html [hunk 2]
```diff
@@ -62,76 +62,61 @@
...
+              <br/><br/><h4>more...</h4>
```
Comment (gregmarra) [inline resources/banter/index.html]: Do you need the `<br/>` tags? Can this be laid out without them?

File: resources/banter/index.html [hunk 3]
```diff
@@ -62,76 +62,61 @@
...
+              <p>Banter is a mentoring program that connects Oliners- who better to help out than someone who's been there?
+There are two types of matches: 
+
+</p><h5>Student/Alumni Matches</h5>
```
Comment (gregmarra) [inline resources/banter/index.html]: Can you move the `</p>` after "matches:"?

**Comment by gregmarra** [comment]
This is your page so I won't be too picky about the HTML in it. If you want me to pull this without changes, I can.

File: resources/banter/index.html (hunk 1 above)
Comment (skier31415) [inline resources/banter/index.html]: done

File: resources/banter/index.html (hunk 2 above)
Comment (skier31415) [inline resources/banter/index.html]: Probably, but I'm not super pro at html.

File: resources/banter/index.html (hunk 3 above)
Comment (skier31415) [inline resources/banter/index.html]: done

**Comment by skier31415** [comment]
Thanks for not being too picky -- I changed what you suggested, though.

**Pull request merged**